meson_options.txt

```diff
@@ -51,3 +51,4 @@ option('metainfo', type: 'boolean', value : true, description : 'install the pro
 option('bash_completion', type: 'boolean', value : true, description : 'enable bash completion')
 option('fish_completion', type: 'boolean', value : true, description : 'enable fish completion')
 option('offline', type: 'boolean', value : true, description : 'enable installing firmware using a pre-boot systemd target')
+option('compat_cli', type: 'boolean', value : true, description : 'enable command line programs provided for compatibility')
```

Comment:
I think this description needs more details

Reply:
God any ideas? that was me being verbose :)

Reply:
List out the programs it creates or why they're still there

Reply:
What about `enable command line programs provided for legacy compatibility`? I'm struggling a bit here.

Reply:
Build legacy applications: $list ?

Reply:
Okay, fixed. Question: is `fwupdtpmevlog` a `compat_cli` program? I've included it here, but it doesn't feel quite right. Maybe we could rename the option to be `extra_binaries` or something? The alternative is that we actually stop installing fwupdtpmevlog and just build it in-src -- as I understood the only real purpose is for debugging the TPM eventlog.

Reply:
At some point we should drop the compat binaries. You're right that tool is debugging only the tpm event log.vi think making it noinst is fine.

Reply:
> At some point we should drop the compat binaries

You have no idea the amount of "paperwork" it would be to rebase RHEL and _not_ provide fwupdate... :)

> tool is debugging only

Done.